dev/small-requirements.txt

```diff
@@ -7,5 +7,5 @@ sqlalchemy>=1.3.21
 pytest==3.2.1
 pytest-cov==2.6.0
 pytest-localserver==0.5.0
-moto==1.3.16
+moto!=2.0.7
```

Comment:
The latest version of `moto` restricts `werkzeug` version to `< 2.0` but `Flask` requires `>= 2.0`.